lib/components/Information/Avatar/index.tsx

```diff
@@ -1,24 +1,33 @@
+import { cn } from "@/lib/utils"
 import {
   Avatar as AvatarComponent,
   AvatarFallback,
   AvatarImage,
 } from "@/ui/avatar"
 import { ComponentProps, forwardRef } from "react"
-
 interface AvatarType {
   alt: string
   src?: string
   size?: ComponentProps<typeof AvatarComponent>["size"]
+  color?: string
 }
 
-export const Avatar = forwardRef<HTMLDivElement, AvatarType>(function Avatar(
-  { src, alt, size },
-  ref
-) {
-  return (
-    <AvatarComponent size={size} ref={ref}>
-      <AvatarImage src={src} alt={alt} />
-      <AvatarFallback>{alt}</AvatarFallback>
-    </AvatarComponent>
-  )
-})
+export const Avatar = forwardRef<HTMLDivElement, AvatarType>(
+  ({ src, alt, size, color }, ref) => {
+    return (
+      <AvatarComponent size={size} ref={ref}>
+        <AvatarImage src={src} alt={alt} />
+        <AvatarFallback
+          className={cn(
+            color && `bg-${color}`,
```

Comment:
This approach is usually quite problematic with tailwind. Tailwind parses the component body statically and determines which classes to keep in the compiled artifact. Since this is changing dynamically, chances are that classes are only available by accident (because other components use them).

The usual approach to do this is to map each color to a class, and this way you also have types for free:

```ts
class colorClassNames = {
  'whatever': 'bg-whatever',
  'foo': 'bg-foo'
}

type AvailableColors = typeof keyof colors
```